### ISSUE TITLE
Meeting with Paul Kilman : Chrome Engineer or Developer Advocate

### ISSUE THREAD

**Issue by aamirp** [URXtech/urx-sdk-web#11]
TLDR : 
Explain why the Chrome fix in V41 onwards breaks the options given by his blog post. 
https://paul.kinlan.me/deep-app-linking-on-android-and-chrome/

@jmilinovich reached out to him and he agreed to talk to us. He will set up some time next week for a call. 
Paul is an advocate for Deeplinking using chrome on Android. Let's use this to discuss anything we might want to talk to him about. Ill try and write up what we are seeing and the exact problem we need Chrome to solve. But random thoughts on the article or anything else on your mind related to chrome on android are welcome. 

@jlturner @andrewlook @jerluc
